.github/workflows/doc-build.yml

```diff
@@ -69,6 +69,7 @@ jobs:
     - name: west setup
       run: |
         west init -l .
+        west update bsim
```

Comment:
I don't get how this is fixing anything. By default the `babblesim` group is disabled, so this will be a nop?

Reply:
this is just fetching the bsim manifest repo (which has submanifest), that one is not under the group

Reply:
No problem with that as long as that works; though, why the `west init -l .` above does not fetch the bsim manifest repo is still a mystery to me.